build_tfm.py

```diff
@@ -65,6 +66,23 @@ def _detect_and_write_tfm_version(tfm_dir, commit):
         _commit_changes(VERSION_FILE_PATH)
 
 
+def _patch_psa_arch_tests(psa_arch_tests_dir, tfm_dir):
+    """
+    Apply patches to psa-arch-tests required by trusted-firmware-m
```

Comment:
Can we not rely on the trusted-firmware-m build system to apply these patches for us?

Reply:
> trusted-firmware-m's build...

Could you put this into the "build_tfm.py: apply patches to psa-arch-tests" commit description? I couldn't understand the reasoning until you explained it further.

Reply:
(I'd expect info about why we can't use TF-M's build system to apply the patches for us in the commit description for "build_tfm.py: apply patches to psa-arch-tests")

Reply:
I will add it

Reply:
Done